tests/IsoCodes/Tests/CreditCardTest.php

```diff
@@ -30,7 +30,6 @@ public function getValidCreditCards()
             array('4903010000000009'),  //Switch
             array('4111111111111111'),  //Visa
             array('6304100000000008'),  //Laser
-            array(6304100000000008),    //Laser
```

Comment:
That depends of the issue finality. See https://github.com/ronanguilloux/IsoCodes/issues/75#issuecomment-215100340.

If you agree that having a integer for a credit card make no sense, so this PR is the way to do.

Otherwise, I'll update or close it.